test/funcfile-docker-rt-tests/testfiles/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:latest
+RUN mkdir /app
+ADd . /app
+WORKDIR /app
+RUN go build -o hello .
+CMD ["/app/hello"]
```

Comment:
you are already in working dir, so just run binary:
```
CMD ["./hello"]
```

Reply:
agree though would prefer '/app/hello' e.g. in case more lines added before it no need to track current workdir

Reply:
Pretty standard practice to do `./`, and it's the opposite. With `./` you don't need to track the workdir.  With `/app/hello`, you need to track the `/app`.

Reply:
updated